src/backend/controllers/userController.js

```diff
@@ -478,11 +478,11 @@ exports.setWager = async (req: Request, res: Response) => {
 		const aDay = 60 * 60 * 24 * 1000;
 
 		// If a user has never made a wager or its been 24hrs since last stipend give them their stipend and update wagerDate
-		if (user.wagerDate == null || (new Date - user.wagerDate) > aDay) {
-			const newBalance = user.money + 5000;
+		if (user.wagerDate == null || (new Date() - user.wagerDate) > aDay) {
+			const newBalance = user.money + 100;
 			user.money = newBalance;
 			user.wagerDate = new Date();
```

Comment:
nit: I know this would never actually happen in real life because we would have to get _insanely_ unlucky, but you should use the same Date object for both this, and the if statement above in case it is exactly a day apart and we cross that deadline in these two lines of code